packages/shared/styles/components/SfHeader.scss

```diff
@@ -1,52 +1,83 @@
-@import '../variables';
+@import "../variables";
 
 $header-mobile-background: $c-light !default;
 $header-mobile-height: 50px !default;
-$header-mobile-justify-content: space-between !default;
+$header-desktop-height: 80px !default;
+
+$header-navigation-padding: 40px !default;
+$header-navigation-justify-content: center !default;
+$header-navigation-list-font-size: $font-size-regular-mobile !default;
+$header-navigation-list-font-weight: 500 !default;
+$header-navigation-list-color: $c-dark !default;
+
+$header-icon-margin: 30px !default;
+$header-icon-cursor: pointer !default;
+
 $header-mobile-align-items: center !default;
+
 $header-mobile-padding-x: 1rem !default;
-$header-tablet-justify-content: flex-start !default;
-$header__logo-width: 21px !default;
+
+$header-mobile-logo-width: 21px !default;
+$header-desktop-logo-width: 34px !default;
+
 $header__logo-height: auto !default;
 
 .sf-header {
   display: flex;
+  align-items: center;
+  width: 100%;
+  margin: {
+    left: auto;
+    right: auto;
+  }
   @media (max-width: $mobile-max) {
+    flex-direction: row;
+    justify-content: $header-navigation-justify-content;
     background: $header-mobile-background;
     height: $header-mobile-height;
-    justify-content: $header-mobile-justify-content;
     align-items: $header-mobile-align-items;
-    padding-left: $header-mobile-padding-x;
-    padding-right: $header-mobile-padding-x;
   }
   @media (min-width: $tablet-min) {
-    justify-content: $header-tablet-justify-content;
+    height: $header-desktop-height;
   }
-  &__column {
-    &--left {
-      order: 1;
-    }
-    &--middle {
-      order: 2;
-    }
-    &--right {
-      order: 3;
-    }
-    &--left,
-    &--right {
-      @media (min-width: $tablet-min) {
-        display: none;
-      }
-    }
+
+  &__navigation {
+    display: flex;
   }
+
+  &__navigation-link {
+    font-size: $header-navigation-list-font-size;
+    font-weight: $header-navigation-list-font-weight;
+    padding-left: $header-navigation-padding;
+    color: $header-navigation-list-color;
+  }
+
+  &__icons {
+    display: flex;
+    margin-right: $header-icon-margin;
+  }
+
+  &__icon {
+    margin-left: $header-icon-margin;
+    cursor: $header-icon-cursor;
+  }
+
+  &__search {
+    margin-left: auto;
+  }
+
   &__logo {
-    width: $header__logo-width;
+    width: $header-mobile-logo-width;
     height: $header__logo-height;
-    padding-right: 0.5rem;
+    padding: 0 0.5rem;
+
+    @media (min-width: $tablet-min) {
+      width: $header-desktop-logo-width;
+    }
   }
-  &__nav,
+  &__navigation,
   &__search,
-  &__icon {
+  &__icons {
     @media (max-width: $mobile-max) {
       display: none;
     }
```

Comment:
please order all properties like here https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

Reply:
Fixed in https://github.com/DivanteLtd/storefront-ui/pull/450/commits/f08adaa0575b53e041449e458718c1c4ea3f129c